Eth-Crowdfund/eth_crowdfund_be
# campaign testing functional

## What's this PR do?
* Create tests for campaign API endpoints

## Where should the reviewer start?
* `eth_crowdfund_api/src/tests/test_campaign.py`

## How should this be manually tested?
* run tests from main folder OR inside the pipenv shell 
* create the test database `createdb eth_crowdfund_api_db_test`
* install pytest and coverage `pip install pytest pytest-cov`
* run command `pytest`
* for coverage run `pytest --cov=src`
  * if that does not work, `pip install pytest-cov`
* if there are errors about modules not being found, make sure they are installed inside AND outside of the pipenv shell

## Any background context you want to provide?
* testing somewhat followed form this article
* https://www.codementor.io/@olawalealadeusi896/building-a-restful-blog-apis-using-python-and-flask-part-3-lx7rt8pfk

## What are the relevant tickets?
close #24
